orchagent/portsorch.cpp

```diff
@@ -5843,6 +5845,40 @@ void PortsOrch::doLagMemberTask(Consumer &consumer)
     }
 }
 
+void PortsOrch::onWarmBootEnd()
+{
+    SWSS_LOG_ENTER();
+
+    m_isWarmRestoreStage = false;
+
+    /* Start dynamic state sync up */
+    refreshPortStatus();
+
+    // Do post boot port initialization
+    for (auto& it: m_portList)
+    {
+        Port& port = it.second;
+
+        if (port.m_type == Port::PHY)
+        {
+            postPortInit(it.second);
+        }
+    }
+}
+
+void PortsOrch::postPortInit(Port& p)
+{
+    SWSS_LOG_ENTER();
+
+    if (gMySwitchType != "dpu")
+    {
+        initializePortBufferMaximumParameters(p);
+    }
+
+    initPortSupportedSpeeds(p.m_alias, p.m_port_id);
+    initPortSupportedFecModes(p.m_alias, p.m_port_id);
```

Comment:
@stepanblyschak these two are called in doPortTask(), so this is no longer needs to be called in doPortTask()?

Reply:
In the original flow ```initPortSupportedSpeeds()``` and ```initPortSupportedFecModes()``` is called twice - when initializing a port and inside doPortTask() and I do not change that.
Both methods are checking if supported speeds and fec modes were queried before and in that case do nothing - https://github.com/sonic-net/sonic-swss/blob/master/orchagent/portsorch.cpp#L3012.

The subsequent calls in doPortTask() might be unnecccesary, but I do not want to change that in this PR.